<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LP-214845099" failures="2" tests="11" name="testcases.DalPulsesTest" time="89.947" errors="0" timestamp="2023-06-30T17:54:41 IST" skipped="0">
  <testcase classname="testcases.DalPulsesTest" name="searchBarElementsTest" time="0.182"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="clickOnSearchIconTest" time="0.116"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="verifySearchElementPageTest" time="0.017">
    <failure message="expected [Toordal- Buy Products Online at Best Price in India - All Categories | Flipkart.com] but found [Flipkart Grocery Store - Buy Groceries Online &amp; Get Rs.1 Deals at Flipkart.com]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Toordal- Buy Products Online at Best Price in India - All Categories | Flipkart.com] but found [Flipkart Grocery Store - Buy Groceries Online & Get Rs.1 Deals at Flipkart.com]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testcases.DalPulsesTest.verifySearchElementPageTest(DalPulsesTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifySearchElementPageTest -->
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="clickableProductTest" time="1.228"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="navigateToPDPTest" time="2.494">
    <failure message="expected [Dehaat Honest Farms Toor/Arhar Dal (Split) Price in India - Buy Dehaat Honest Farms Toor/Arhar Dal (Split) online at Flipkart.com:] but found [Dehaat Honest Farms Toor/Arhar Dal (Split) Price in India - Buy Dehaat Honest Farms Toor/Arhar Dal (Split) online at Flipkart.com]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Dehaat Honest Farms Toor/Arhar Dal (Split) Price in India - Buy Dehaat Honest Farms Toor/Arhar Dal (Split) online at Flipkart.com:] but found [Dehaat Honest Farms Toor/Arhar Dal (Split) Price in India - Buy Dehaat Honest Farms Toor/Arhar Dal (Split) online at Flipkart.com]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testcases.DalPulsesTest.navigateToPDPTest(DalPulsesTest.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- navigateToPDPTest -->
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="addItemTest" time="0.252"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="addQuantityTest" time="10.251"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="cart" time="0.192"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="addQuantity1Test" time="20.251"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="placeOrderTest" time="14.067"/>
  <system-out/>
  <testcase classname="testcases.DalPulsesTest" name="removeProductTest" time="40.897"/>
  <system-out/>
</testsuite> <!-- testcases.DalPulsesTest -->
